### PR TITLE
adding openHelp method to client

### PR DESCRIPTION
**Pull request by Blackbaud-CoreyArcher**

### CODE REVIEW

File: src/addin/client-interfaces/addin-client-open-help-args.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/**
+ * Interface for launching the help tab from an add-in.
+ */
+export interface AddinClientOpenHelpArgs {
+
+  /**
+   * The name of the help key to be launched.
+   */
+  helpKey?: string;
```
Comment (Blackbaud-BenLambert) [inline src/addin/client-interfaces/addin-client-open-help-args.ts]: any reason not to require helpKey?  I.e., why allow null?

Comment (Blackbaud-BenLambert) [inline src/addin/client-interfaces/addin-client-open-help-args.ts]: also - need to document this in the readme

Comment (Blackbaud-CoreyArcher) [reply src/addin/client-interfaces/addin-client-open-help-args.ts]: My mistake, actually. I was intending to follow the navigate pattern, which is non-nullable, but I think I looked at the openModalArgs IF by mistake, which was nullable. I will fix it and update.

Comment (Blackbaud-CoreyArcher) [reply src/addin/client-interfaces/addin-client-open-help-args.ts]: updated. Also fixed above helpKey.

**Pull request merged**